capabilities/vcs/single-file-content/profile.supr

```diff
@@ -1,37 +1,75 @@
 """
-VCS File Content
+Single File Content
 Retrieve the raw content of single file from a version control system repository.
 """
 name = "vcs/single-file-content"
-version = "1.0.2"
+version = "1.0.3"
 
 """
-Get Single File Content
+Single File Content
 Get the content of the specified file.
 """
 usecase SingleFileContent {
   input {
-    "Owner of the repository"
+    """
+    Owner
+    Owner of the repository
+    """
     owner! string!
 
-    "Repository name"
+    """
+    Repo
+    Repository name
+    """
     repo! string!
 
-    "Path to file at repository"
+    """
+    Path
+    Path to file at repository
+    """
     path! string!
 
-    "Branch eg. main"
+    """
+    Ref
+    Branch eg. main
+    """
     ref! string!
   }
 
   result {
-    "Size of content according to used provider"
+    """
+    Size
+    Size of content according to used provider
+    """
     size! number!
 
-    "Content of file"
+    """
+    Content
+    Content of file
+    """
     content! string!
 
-    "Used encoding"
+    """
+    Encoding
+    Used encoding
+    """
     encoding! string!
   }
+  error {
```

Comment:
Adding error to profile should result in minor version update as error handling changes in consumers applications.